coalib/output/Logging.py

```diff
@@ -32,7 +32,7 @@ def get_num_calls_for_level(cls, level):
         return cls._call_counter[level]
 
 
-def configure_logging():
+def configure_logging(no_color=False):
```

Comment:
Although coala uses a `no-color` argument, imo the internal API doesn't have to follow the nomenclature 1:1 ;)